infrastructure/helm/bailo/templates/bailo/bailo.configmap.yaml

```diff
@@ -191,6 +192,10 @@ data:
           registry: '{{ .Values.minio.registryBucket }}',
         },
       },
+
+      inference: {
+        tokenHeader: '{{ .Values.inference.tokenHeader }}'
```

Comment:
I'm assuming the is sensitive information and as such should be included in the `custom-environment-variables` and be populated via a secret.